internal/core/app_release.go

```diff
@@ -58,10 +58,12 @@ func (a *App) Release(ctx context.Context, target *pb.Deployment) (
 			"err", err)
 	}
 
+	unimplemeneted := false
 	c, err := a.createReleaser(ctx, &evalCtx)
 	if status.Code(err) == codes.Unimplemented {
 		c = nil
 		err = nil
+		unimplemeneted = true
```

Comment:
typo 😱 
😁

Reply:
Good catch!! Will be fixed via https://github.com/hashicorp/waypoint/pull/1801/commits/a9ef3f27783b971fc08e08f00dd0b64347e212be in the app poll feature PR 🙏🏻